recipes/watch.mdwn

```diff
@@ -62,6 +62,26 @@ local cmus, cmus_timer = awful.widget.watch(
 )
 ```
 
+## iostat
+
+```lua
+-- disk I/O using iostat from sysstat utilities
+local iotable = {}
+local iostat = awful.widget.watch("iostat -dk", 2, -- in Kb, use -dm for Mb
+    function(widget, stdout)
+        for line in stdout:match("(sd.*)\n"):gmatch("(.-)\n") do
+            local device, tps, read_s, wrtn_s, read, wrtn =
+            line:match("(%w+)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)")
+            --                  [1]  [2]     [3]     [4]   [5]
```

Comment:
sorry for the late review, why not just 

```lua
iotable[device] = {
  tps=tps,
  read_s=read_s
```

?

Reply:
What for?

Reply:
to avoid the comment on line 75 and write it more readable:
`iotable.sda.read_s` instead of `iotable["sda"][2]`

Reply:
OK, change it yourself if you want. I'm too lazy to open another PR for a couple of lines. 😺